index.md

```diff
@@ -0,0 +1,364 @@
+
+4. In the ticket description, make sure you add:
+   1. Test goal
+   2. The pre-conditions
+   3. The steps needed
+
+5. Define the priority for writing down this test.
+   See the [Priorities](#priorities) session for more details.
+
+You can use the Test Create Priorities to define the ticket priority.
+
+## Labels
+
+Labels are used to quickly find tickets, create queries, boards, and dashboards.
+The labels here should all be lower case and words separated by a underscore (`_`).
+Please, avoid creating labels that can be de-coupled into two other labels.
+For example: instead of `lsstcam_science`, use `lsstcam` and `science`.
+Or, even better, just use `science`, since `lsstcam` is the default camera.
+
+| **Label**           | **Description**                                                                 |
+|---------------------|---------------------------------------------------------------------------------|
+| `create`            | Used for the creation of a test case or JSON file.                              |
+| `update`            | Used for updating a test case or JSON file.                                     |
+| `json_block`        | Used when a JSON file is required.                                              |
+| `on_sky/not_on_sky` | Used for on-sky or not-on-sky tests.                                            |
+| `daytime_test`      | Used for tests that could be performed during the daytime.                      |
+| `parallelize`       | Used for tests that have potential for being parallelized.                      |
+|---------------------|---------------------------------------------------------------------------------|
+| `aos_commissioning` | Used for tests associated with the Active Optics System                         |
+| `LSSTCam`           | Used for tests aiming to check functionality of requirements associated with LSSTCam |
+| `calibration`       | Used for tests related to the calibration system                                |
+| `image_quality`     | Used for tests associated with image quality.                                   |
+| `science`           | Used for Science Observations programs                                          |
+| `stray_light`       | Used for tests associated with stray light investigations.                      |
+| `standard_procedures` | Used for procedures that are represented as test cases.                       |
+|-------------------------------------------------------------------------------------------------------|
+
+Each row in the second part of the table above corresponds to a Quick Filter in the [Blocks Creation Status](https://rubinobs.atlassian.net/jira/software/c/projects/BLOCK/boards/259) board.
+
+## Priorities
+
+Each of these steps has a definition of priority.
+We should have an agreement on each of them. Here is the first proposal.
+When it comes to priorities, we must agree on the following points:
```

Comment:
I know this was copied from a proposal in confluence, but it's probably easiest to just set the rules here and speak more directly. The four priorities you list below are enough (critical, high, standard, low).